doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc

```diff
@@ -1,17 +1,26 @@
 [id='rn-kogito-productize-features-ref']
 = Kogito productized features in {PRODUCT}
 
-{PRODUCT} {ENTERPRISE_VERSION} provides the following Kogito based applications that you can use to design Business Process Model and Notation (BPMN) process models and Decision Model and Notation (DMN) decision models using graphical modelers:
+== Support for {KOGITO} microservices in {PRODUCT}
 
-* VSCode extension including BPMN, DMN, and test scenario editor
-* Standalone embeddable DMN and BPMN editors
-* Standalone embeddable DMN viewer
+{PRODUCT} now provides support for the following {KOGITO} microservices that you can use to build cloud-native applications that adapt to your business domain and tooling:
 
-Customers who own the {PRODUCT} can access these new applications or extensions without acquiring a new subscription.
+* Layering on https://quarkus.io/[Quarkus] or https://spring.io/projects/spring-boot[Spring Boot] to enable developers to run {KOGITO} microservices on their preferred cloud-native application development stack
+* Automated decision-making capabilities based on rule units to run within defined business constraints
+* Example applications containing various types of {KOGITO} decision microservices on https://quarkus.io/[Quarkus] or https://spring.io/projects/spring-boot[Spring Boot] to help you develop your own applications
 
-== Additional resources
+== Support for {KOGITO} JVM Quarkus runtime
 
-* {URL_DEVELOPING_DECISION_SERVICES}#assembly-dmn-models[_{DMN_MODELS}_]
-* {URL_DEVELOPING_PROCESS_SERVICES}#assembly-designing-business-processes[_{DESIGNING_BUSINESS_PROCESSES}_]
-* {URL_GETTING_STARTED}#assembly-getting-started-decision-services[_{GETTING_STARTED_DECISION_SERVICE}_]
-* https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-red-hat-business-automation-bundle[_Red Hat Business Automation Bundle_]
+{PRODUCT} now supports {KOGITO} JVM Quarkus runtime images.
+
+== Support for {KOGITO} builder images for Qarkus and SpringBoot
+
+{PRODUCT} now includes {KOGITO} builder images for Quarkus and SpringBoot.
+
+ifdef::PAM[]
+
+== Support for {KOGITO} operator on {OPENSHIFT} 4
```

Comment:
This title does not seem to match description?  The description doesn't have anything to do with Kogito, but is about integrating RHPAM applications (on Kie-server) with Kafka, that is now also available when deploying on OpenShift (but this already included lower in section 3.9.1), so seems description needs to be updated here to reflect the title?